Detectors/ITSMFT/ITS/macros/test/CheckClusters.C

```diff
@@ -5,17 +5,25 @@
   #include <TFile.h>
```

Comment:
I think instead of modifying CheckClusters to build a topologies map, it is better to have a dedicated macro for that. Could you please clone it from this macro?